src/EditorFeatures/CSharpTest2/Recommendations/InKeywordRecommenderTests.cs

```diff
@@ -301,5 +301,126 @@ void M()
     {
         var q = from Int32 $$");
         }
+
+        [Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.ReadonlyReferences)]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InWillPopUpAsParameterModifier_Methods()
```

Comment:
I think the naming convention here is more TestAsParameterModifier..."will pop up" is a bit verbose.